front/src/modules/rollingStock/components/RollingStockSelector/SearchRollingStock.tsx

```diff
@@ -48,8 +51,22 @@ function rollingStockPassesSearchedStringFilter(
   ].some(includesSearchedString);
 }
 
```

Comment:
It is not obvious to read. 
please rename filterList:Filters -> filters:Filters
be more explicit about the locked param (inline doc ?)

Reply:
Something is either locked or unlocked. If you absolutely need to use 2 different fonctions which will cause errors, please explain why

Reply:
done, comment added for interface